.github/workflows/javabuild.yaml

```diff
@@ -0,0 +1,16 @@
+name: Java Build
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build with Maven
+        run: mvn package --file pom.xml
```

Comment:
I believe this needs to point to the pom file exactly. Right now the build is failing (but at least we know the checks are running correctly!).
Try changing this to something like mvn package --file portfolio/pom.xml